src/Riak/TimeSeries/Cell.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Basho\Riak\TimeSeries;
+
+/**
+ * Data structure for Cells of a TimeSeries row
+ *
+ * @author Christopher Mancini <cmancini at basho d0t com>
+ */
+class Cell
+{
+    const STRING_TYPE = 'varchar';
+    const INT_TYPE = 'sint64';
+    const DOUBLE_TYPE = 'double';
+    const BOOL_TYPE = 'boolean';
+    const TIMESTAMP_TYPE = 'timestamp';
+
+    protected $name;
+    protected $value = null;
+    protected $type = null;
+
+    public function __construct($name)
+    {
+        $this->name = $name;
+    }
+
+    /**
+     * @return string
+     */
+    public function getType()
+    {
+        return $this->type;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    /**
+     * @return string
+     */
+    public function getValue()
+    {
+        return $this->value;
+    }
+
+    /**
+     * @param string $value
+     */
+    public function setValue($value = null)
```

Comment:
Yes, in fact I think thats how variables are stored at a C level, allowing for type juggling.